evap/staff/views.py

```diff
@@ -1453,11 +1453,15 @@ def evaluation_login_key_export(_request, evaluation_id):
     return response
 
 
-def get_evaluation_and_contributor_textanswer_sections(evaluation, filter_textanswers):
-    TextAnswerSection = namedtuple(
-        "TextAnswerSection", ("questionnaire", "contributor", "label", "is_responsible", "results")
-    )
+TextAnswerSection = namedtuple(
+    "TextAnswerSection", ("questionnaire", "contributor", "label", "is_responsible", "results")
+)
+
 
+def get_evaluation_and_contributor_textanswer_sections(
+    evaluation: Evaluation,
+    textanswer_filter: Q,
```

Comment:
`optional[Q]`? Below the code checks for `if textanswer_filter is not None`

Reply:
ah, actually the check was not needed - weird that mypy / pylint didnt catch it :thinking: